utility_analysis_new/parameter_tuning.py

```diff
@@ -301,13 +300,91 @@ class TuneResult:
           feasible.
         options: input options for tuning.
         contribution_histograms: histograms of privacy id contributions.
-        utility_analysis_runs: the results of all utility analysis runs that
+        utility_analysis_parameters: contains tune parameter values for which
```

Comment:
That's because during parameter tuning, utility analysis is performed in order to find better results. I've updated tune() docstring with explanation.